README.md

```diff
@@ -13,6 +13,47 @@ $ sudo apt install open-fprintd fprintd-clients python3-validity
 $ fprintd-enroll
 ```
 
+### Error situations
+If `fprintd-enroll` returns with `list_devices failed:`, you can check
+the logs of the `python3-validity` daemon using `$ sudo systemctl status python3-validity`.
+If it's not running, you can enable and/or start it by substituting `status` with `enable` or `start`.
+
+It `systemctl status python3-validity` complains about errors on startup, you may need to factory-reset the fingerprint chip. Do that like so:
+```
+$ sudo systemctl stop python3-validity
+$ sudo validity-sensors-firmware
+$ sudo python3 /usr/share/python-validity/playground/factory-reset.py
+
+# At this point you may or may find the device busy, depending on
+# how systemctl plays along. Kill offending processes if necessary.
+
+$ sudo systemctl start python3-validity
```

Comment:
`factory-reset.py` causes the USB device to be disconnected and then reconnected back. I think `udev` rule will start the `python3-validity` automatically whenever it is reconnected even if you've stopped it manually before, so this line may not be required. But no harm in starting it twice anyway.